docs/README.md

```diff
@@ -0,0 +1,116 @@
+
+This is done by adding the xacro macro `ign_ros2_control` (which is integrated 
+from main Panda URDF file) and add the Gazebo plugin `IgnitionROS2ControlPlugin`
+that parses the `ros2_control` tags and loads the appropriate hardware interfaces and controller manager
+
+* https://github.com:drejOrsula/panda_ign_moveit2/blob/master/panda_description/urdf/panda.gazebo#L8-L16
+
+### How to connect MoveIt 2 with the Gazebo ros2_control plugins
+
+The example launches MoveIt using igniton_ros2_control:
+
+* [default.launch.py](../launch/default.launch.py#L81-L94)
+
+This snippet is calling the main [Panda MoveIt configuration launch file](https://github.com/AndrejOrsula/panda_ign_moveit2/blob/master/panda_moveit_config/launch/move_group.launch.py):
+
+* Panda main MoveIt 2 move_group invocation:
+  * https://github.com/AndrejOrsula/panda_ign_moveit2/blob/5679c0a97d5e34199271ed7abf02cf303d0e8fb9/panda_moveit_config/launch/move_group.launch.py#L248-L264
+
+* Controller manager
+  * https://github.com/AndrejOrsula/panda_ign_moveit2/blob/5679c0a97d5e34199271ed7abf02cf303d0e8fb9/panda_moveit_config/launch/move_group.launch.py#L309-L324
+
+### How this example declares the target pose for the manipulation
+
+This repository uses a `Gazebo PosePublisher` plugin to indicate the target pose for the manipulation:
+* [follow_target.sdf](../ign_moveit2_examples/worlds/follow_target.sdf#L99-L104)
+
+Translate the pose from Gazebo to ROS using the `ros_ign_bridge`:
+* [world_follow_target.launch.py](../launch/worlds/world_follow_target.launch.py#L61-L76)
+
+The /target_pose is subscribed to by the ROS C++ node that communicates to MoveIt
+* [ex_follow_target.cpp](../examples/cpp/ex_follow_target.cpp#L28-L57)
+
+### Other important ROS 2 packages
+
+#### Robot state publisher
+
+The Robot State Publisher is a node and a class to publish
+the state of a robot to tf2. At startup time, Robot State Publisher is 
+supplied with a kinematic tree model (URDF) of the robot. It then subscribes 
+to the joint_states topic (of type sensor_msgs/msg/JointState) to get
+individual joint states. More information in the
+[ROS2 tutorial](https://docs.ros.org/en/galactic/Tutorials/Intermediate/URDF/Using-URDF-with-Robot-State-Publisher.html)
+
+* https://github.com/ros/robot_state_publisher
+
+#### TF2 
+
+tf2 is the second generation of the transform library, which lets the user
+keep track of multiple coordinate frames over time. tf2 maintains the 
+relationship between coordinate frames in a tree structure buffered in 
+time, and lets the user transform points, vectors, etc between any 
+two coordinate frames at any desired point in time.
+
+* http://wiki.ros.org/tf2
+
+
+## ROS 2 Topics and Nodes
+
+![ROS 2 Graph of the demo](ros_rqt_graph.png)
+
+Colors legend for nodes:
+ * Green: Gazebo special nodes for ROS integration
+ * Purple: pose target created by the own example code in this repo
+ * Blue: nodes create and handle by `ros2_control` invoked from `ignition_ros_control`
```

Comment:
```suggestion
 * Blue: nodes created and handled by `ros2_control` invoked from `gz_ros_control`
```

Reply:
If we keep it ignition, that's fine, but it should be `ign_ros2_control` for consistency.